R/getIgraph.R

```diff
@@ -0,0 +1,83 @@
+#' Igraph plotting function for miic
+#' @description This functions returns an igraph object built from the results
+#' of the miic main function.
+#'
+#' @details
+#' Edges attributes are passed to the igraph graph and can be accessed with
+#' e.g. \code{E(g)$partial_correlation}. See \code{\link{miic}} for more
+#' details on edge parameters. By default, edges are colored according to the
+#' partial correlation between two nodes conditioned on the conditioning set
+#' (negative is blue, null is gray and positive is red) and their width is
+#' based on the conditional mutual information minus the complexity cost.
+#'
+#' @param miic.res [a miic graph object]
+#' The graph object returned by the miic execution.
+#' 
+#' @return An igraph graph object.
+#' 
+#' @export
+#' @useDynLib miic
+#' 
+#' @seealso
+#' \code{\link{miic}} for details on edge parameters in the returned object, 
+#' \code{\link[igraph]{igraph.plotting}} for the detailed description of the
+#' plotting parameters and \code{\link[igraph]{layout}} for different layouts.
+#' 
+#' @examples
+#' \dontrun{
+#' library(miic)
+#' library(igraph)
+#' 
+#' g = getIgraph(miic.res)
+#' plot(g) # Default visualisation
```

Comment:
```suggestion
#' 
#' g = miic.export(miic.res, target = "igraph")
#' igraph::plot(g) # Default visualisation
```

Reply:
OK for the generic API

I thought about it and I think we should have "plot" in at least one of the new function names for newer users. I'll mark this PR as draft and make a few changes